dist/stat_data.py

```diff
@@ -222,7 +222,6 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_fail', 'pages selected for eviction unable to be evicted'),
     CacheStat('cache_eviction_force', 'pages evicted because they exceeded the in-memory maximum count'),
     CacheStat('cache_eviction_force_delete', 'pages evicted because they had chains of deleted items count'),
-    CacheStat('cache_eviction_force_delete_time', 'pages evicted because they had chains of deleted items time (usecs)'),
```

Comment:
I think this change is worthwhile, but I'm inclined to approach it a bit differently. I'd like to end up with:

`pages queued for eviction because they had chains of deleted items` (I believe this matches the code change you made).
`pages evicted immediately that were clean` (along with matching time spent stat)
`pages evicted immediately that were dirty` (along with matching time spent stat)

I'd be fine with replacing `immediately` with `force`, or something more verbose describing `due to reasons other than normal cache management`

The reason I care is that tracking all pages that are evicted inline is valuable.

Reply:
@agorrod, I've pushed this change in c1e7a87.

It seemed cleanest to move the forcible eviction statistics out of `__wt_page_release_evict` and into `__wt_evict` because the latter function already knows if the evicted page was clean or dirty, and updates statistics based on that fact. The one thing this changes is we won't include forcible eviction requests resulting in in-memory splits as failed or successful forcible evictions.

Let me know if you want further changes or I've misunderstood something.